core/arch/arm/plat-vexpress/fvp_spmc_pm.c

```diff
@@ -1,24 +1,15 @@
 // SPDX-License-Identifier: BSD-2-Clause
 /*
- * Copyright (c) 2019, Arm Limited
+ * Copyright (c) 2019-2021, Arm Limited
  */
 
 #include <arm.h>
 #include <ffa.h>
-#include <initcall.h>
-#include <keep.h>
 #include <kernel/boot.h>
-#include <kernel/interrupt.h>
-#include <kernel/misc.h>
 #include <kernel/panic.h>
-#include <mm/core_memprot.h>
-#include <mm/core_mmu.h>
-#include <platform_config.h>
-#include <sm/psci.h>
-#include <stdint.h>
-#include <string.h>
-#include <trace.h>
+#include <kernel/thread.h>
 
+#ifdef CFG_ARM_TRUSTED_FW_2_4
```

Comment:
Does this work with the QEMU v8 setup we have too? Is it possible to keep the AArch32 compatibility?
I suppose this should be marked as deprecated too since we'd like to get rid of it at some time.

Reply:
Reverted to original version of the patch, no longer relevant.